cli/wallet/multisig.go

```diff
@@ -31,9 +29,6 @@ func signStoredTransaction(ctx *cli.Context) error {
 		return cli.Exit(err, 1)
 	}
 
-	if !addrFlag.IsSet {
-		return cli.Exit("address was not provided", 1)
-	}
```

Comment:
You change the behaviour here. Mark "address" flag from `signFlags` as required.

Reply:
But if --address is not provided then the default one will be used in options.GetAccFromContext(ctx). So that is why I marked it in usage as optional and removed this check. So the question is what is the correct behaviour?